app/Http/Requests/Backend/UpdateUserRequest.php

```diff
@@ -23,7 +24,7 @@ public function authorize()
     public function rules()
     {
         return [
-            'password' => 'required|min:3',
+            'password' => 'nullable|min:3',
```

Comment:
if remove it will catch validate because null is less than 3
![screenshot from 2017-08-16 11-20-52](https://user-images.githubusercontent.com/24862074/29347200-0b73e2cc-8275-11e7-81f8-bf01d46b6671.png)